### ISSUE TITLE
Blank album appears in Add to Album modal

### ISSUE THREAD

**Issue by Snowknight26** [immich-app/immich#9657]
### The bug

If you go to the Sharing page, start creating an album, and then close out of it, a blank album appears in the Add to Album modal.

### The OS that Immich Server is running on

Windows 10 22H2 (19045.4291)

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Go to Sharing
2. Click Create Album in the top right
3. Click the back ("Close") icon or press escape
4. Go to Photos
5. Select a photo
6. Click the plus ("Add to...") icon in the top right
7. Select Add to Album
```

### Additional information

![image](https://github.com/immich-app/immich/assets/5608270/149ca1bf-f2de-4592-a3b3-e81517fb8dd3)

Refreshing the page makes and then doing steps 4-7 makes them go away, so they seem to be temporary albums in memory instead of in the database.

**Comment by aviv926** [comment]
I can't reproduce the problem (Android, latest version.)
Are you using a reverse proxy?

**Comment by Snowknight26** [comment]
No reverse proxy, but this only affects the web version.

**Comment by aviv926** [comment]
Yes, my bad, I didn't notice that you mentioned it on the Web

**Comment by remshams** [comment]
Navigating to the albums page (after adding it in sharing) also "fixes" the bug. You can quickly see the album popping up and then it's gone (also from the context menu when selecting a photo).

**Comment by Snowknight26** [comment]
Fixed by #9771.
